recipes/cxxtest/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "cxxtest" %}
+{% set version = "4.4" %}
+{% set sha256 = "1c154fef91c65dbf1cd4519af7ade70a61d85a923b6e0c0b007dc7f4895cf7d8" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+     
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+    url: https://github.com/CxxTest/cxxtest/releases/download/4.4/cxxtest-4.4.tar.gz
+
+build:
+    noarch: python
+    number: 0
+
+requirements:
+    host:
+        - python
+    run:
+        - python
+
+test:
+    source_files:
+        - doc/examples/MyTestSuite1.h
+
+about:
+    home: http://cxxtest.com/
+    license: LGPLv3
+    license_file: COPYING
+    summary: 'CxxTest Unit Testing Framework.'
+    description: |
+      CxxTest is a unit testing framework for C++ that is similar in spirit to
+      JUnit, CppUnit, and xUnit. CxxTest is easy to use because it does not
+      require precompiling a CxxTest testing library, it employs no advanced 
+      features of C++ (e.g. RTTI) and it supports a very flexible form of test
+      discovery.
+    doc_url: http://cxxtest.com/guide.html
+    dev_url: https://github.com/CxxTest/cxxtest
+
+extra:
+    recipe-maintainers:
+      - swainn
```

Comment:
@swainn, are you ok being listed as a maintainer here?

Reply:
Yes.